l10n_ch_invoice_reports/data/reports.xml

```diff
@@ -1,39 +1,15 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <odoo noupdate="0">
     <report
-        id="account_move_isr_report"
+        id="account_move_payment_report"
         model="account.move"
         report_type="qweb-pdf"
-        string="Invoice with ISR"
-        name="l10n_ch_invoice_reports.invoice_isr_report_main"
-        file="l10n_ch_invoice_reports.invoice_isr_report_main"
-        attachment_use="False"
-        attachment=""
+        string="Invoice with payment slip(s)"
+        name="l10n_ch_invoice_reports.account_move_payment_report"
+        file="l10n_ch_invoice_reports.account_move_payment_report"
+        attachment_use="True"
+        attachment="invoice_with_payslip.pdf"
```

Comment:
```suggestion
        attachment="invoice_'+(object.name or '')+'_with_payslip.pdf"
```
name assigned when invoice confirmed so sometimes it can be empty

Reply:
@i-vyshnevska  on posted invoice:
![image](https://user-images.githubusercontent.com/59824990/183379880-e12a968e-6409-4e07-9710-e0b97f1ee4f9.png)


btw. can be attachment's name translated? they would like to have it in different languages depending on customer...

Reply:
"'invoice_'+(object.name or '')+'_with_payslip.pdf'"
quotes

Reply:
this string isn't taken properly, doesn't matter what i am doing. Maybe there is some trick with report action (which we dont call)
![image](https://user-images.githubusercontent.com/59824990/183408494-a6477a7d-6dc5-4597-80e3-8ccc5a34bcc2.png)